recipes/flac/all/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(cmake_wrapper)
+
+if(EXISTS "${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
+  include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
+else()
+  include(conanbuildinfo.cmake)
+endif()
```

Comment:
```suggestion
include(conanbuildinfo.cmake)
```

Reply:
Why this change? We added this to our templates because it seems necessary  for some workflows